prog/dftb+/lib_timedep/linrespcommon.F90

```diff
@@ -172,11 +177,13 @@ subroutine getNorb_r(nxov, win, getij, homo, no, nv)
     no = 0
     nv = 0
 
+    !$OMP PARALLEL DO DEFAULT(SHARED) PRIVATE(ia,ii,jj) SCHEDULE(RUNTIME) REDUCTION(+:nv,no)
```

Comment:
General remark: The iteration variable for which the `$OMP PARALLEL DO` is defined is private by default, so it does not need to be listed.
Side note: I am wondering, whether we should switch to use `DEFAULT(NONE)` at some point to reduce chances of having a shared variable due to forgetting of list it as private...